NuGet.config

```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
     <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
+    <add key="DotNet-Core" value="https://myget.org/f/dotnet-core/api/v3/index.json" />
```

Comment:
Should we mirror these packages instead? We did this once before where we were relying on CoreCLR \ Roslyn packages from their feeds. However we're then affected by their package deletion policies which aren't particularly generous. That said, we already mirror packages from dotnet-core, so it would be adding dotnet-cli to the list.